packages/waci-pex/src/types/ssi-qr-code-provider-types.ts

```diff
@@ -0,0 +1,46 @@
+import { IAgentContext, IPluginMethodMap } from '@veramo/core'
+import {SIOP} from "@sphereon/did-auth-siop/";
+import {QrPropValue} from "../agent/qr-utils/QrPropValue";
+
+export interface SsiQrCodeProviderTypes extends IPluginMethodMap {
+  ssiQrCode(qrPropValue: QrPropValue, context: IRequiredContext): Promise<JSX.Element>
+}
+
+export enum QRType {
+  AUTHENTICATION = 'auth',
+}
+
+export enum QRMode {
+  DID_AUTH_SIOP_V2 = 'didauth',
+}
+
+export interface QRContent {
+  state: string
+  nonce: string
+  qrValue: string
+}
+
+export enum AcceptValue {
+  // DID_COMM_V2 = "didcomm/v2",
```

Comment:
can be removed. if we want to add it later, we can add it later

Reply:
Want to keep it for hint about the extension point.

Reply:
so when are we extending this?

Reply:
I do not have answer to that.